code/game/gamemodes/events/biomass.dm

```diff
@@ -27,13 +27,11 @@
 	if (!W || !user || !W.type)
 		return
 
-	switch(W.type)
+	switch(W.type) //This is absolutely terrible

```

Comment:
I do not care about the snowflake's code original function.

It was bad and it needs to be sent to the hellscape.